src/Queue/Adapter/Swoole.php

```
@@ -42,7 +85,8 @@
 
     public function workerStop(callable $callback): self
     {
-        $this->pool->on('WorkerStart', function (Pool $pool, string $workerId) use ($callback) {
+        $this->onStop = $callback;
+        $this->pool->on(Constant::EVENT_WORKER_STOP, function (Pool $pool, string $workerId) use ($callback) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$pool'. (Unused Code Rules)

(UnusedFormalParameter)